source/_integrations/zwave_js.markdown

```diff
@@ -515,6 +515,8 @@ There is no automatic migration wizard for the `ozw` integration. Please follow
 
 2) Remove the Z-Wave integration from Home Assistant: Configuration --> Integrations --> Z-Wave (or OpenZWave) --> Press the three dots and click Delete.
 
+    <div class='note info'>If you have configured Z-Wave manually, make sure to also remove the `zwave:` section from your configuration.yaml.</div>
```

Comment:
Note sure how we can get the `zwave:` to render as a codeblock. It doesn't seem to work when the note div is indented in a numbered list.

Reply:
```suggestion
    <div class='note info'>
    
    If you have configured Z-Wave manually, make sure to also remove the `zwave:` section from your `configuration.yaml`.
    
    </div>
```

The Markdown parser needs an empty line in order to switch HTML/Markdown context.